netbox/dcim/forms/bulk_edit.py

```diff
@@ -449,14 +459,14 @@ class RackBulkEditForm(NetBoxModelBulkEditForm):
         FieldSet('status', 'role', 'tenant', 'serial', 'asset_tag', 'rack_type', 'description', name=_('Rack')),
         FieldSet('region', 'site_group', 'site', 'location', name=_('Location')),
         FieldSet(
-            'form_factor', 'width', 'u_height', 'desc_units', 'airflow', 'outer_width', 'outer_depth', 'outer_unit',
-            'mounting_depth', name=_('Hardware')
+            'form_factor', 'width', 'u_height', 'desc_units', 'airflow', 'outer_width', 'outer_height', 'outer_depth',
+            'outer_unit', 'mounting_depth', name=_('Hardware')
```

Comment:
We should move the outer dimension fields to a separate fieldset like we did for RackTypeBulkEditForm.